distro/README.md

```diff
@@ -0,0 +1,79 @@
+# Package `github.com/signalfx/splunk-otel-go/distro`
+
+This package provides a Splunk distribution of the OpenTelemetry Go SDK. It is
+designed to provide an SDK properly configured to be used with the Splunk
+platform out-of-the-box.
+
+## Getting Started
+
+The main entry point for the package is the [`Run`][] function. Use this
```

Comment:
nit: I think you can change
```
[`Run`][]
```
to 
```
[`Run`]
```
but you would have to test it 😄 

this is at least what I was always familiar with and is used here: https://raw.githubusercontent.com/signalfx/splunk-otel-go/main/CHANGELOG.md